.github/workflows/build_commit.yaml

```diff
@@ -0,0 +1,47 @@
+name: build-apps
+
+on:
+  push:
+    branches:
+      - master
+jobs:
+  apk:
+    name: Generate APK
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '11'
+      - uses: subosito/flutter-action@v1
+        with:
+          flutter-version: '3.3.4'
+          channel: 'stable'
+      - run: mkdir assets/json
+      - run: echo $SECRETS_FILE_CONTENT | base64 -d > assets/json/config.json
+        env:
+          SECRETS_FILE_CONTENT: ${{ secrets.SECRETS_FILE_CONTENT }}
+      - run: flutter pub get
+      - run: echo $GOOGLE_SERVICES_FILE_CONTENT | base64 -d > android/app/google-services.json
+        env:
+          GOOGLE_SERVICES_FILE_CONTENT: ${{ secrets.GOOGLE_SERVICES_FILE_CONTENT }}
+      - name: Build APK
+        run: flutter build apk --flavor prod -t lib/main_prod.dart --release
+      - name: Create Github Release
+        id: create-new-release
+        uses: actions/create-release@v1
```

Comment:
<img width="842" alt="image" src="https://user-images.githubusercontent.com/64923802/198559569-08a21369-fe2f-420c-8736-b02c841ba62e.png">

let's use the maintained repository

Reply:
Fixed.